src/transformState.js

```diff
@@ -5,7 +5,25 @@
  * @param {Object[]} actions
  */
 function transformState(state, actions) {
-  // write code here
+  for (const x of actions) {
```

Comment:
для змінних потрібно задавати відповідні назви:
```suggestion
  for (const action of actions) {
```